### PR TITLE
Install pydocstyle for checking docstrings

### PR DESCRIPTION
**Pull request by fridex**
Signed-off-by: Fridolin Pokorny <fridolin.pokorny@datadoghq.com>

Related: https://github.com/theupdateframework/python-tuf/pull/2282 https://github.com/theupdateframework/python-tuf/pull/2281

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -154,3 +154,7 @@ module = [
   "securesystemslib.*",
 ]
 ignore_missing_imports = "True"
+
+[tool.pydocstyle]
+inherit = false
+ignore = "D400,D209,D415,D213,D205,D202,D107,D407,D413,D212,D104,D406,D105,D411,D401,D200,D203"
```
Comment (lukpueh) [inline pyproject.toml]: This changes with #2281 #2282, right?

Comment (fridex) [reply pyproject.toml]: Yes, reduce the listing over time. In the mean time, not to introduce new violations.

Comment (lukpueh) [inline pyproject.toml]: https://github.com/theupdateframework/python-tuf/pull/2281 has been merged. This should work now:
```suggestion
ignore = "D400,D415,D213,D205,D202,D107,D407,D413,D212,D104,D406,D105,D411,D401,D200,D203"
```

Comment (lukpueh) [reply pyproject.toml]: Please update and we can merge.

**Comment by jku** [comment]
This and #2289 will conflict a little bit -- I'm happy to leave #2289 waiting and fix the conflict there but if you're not working on this now, we can switch the other way round.

**Comment by lukpueh** [comment]
Looks like pydoc caught something. :)

**Review by jku — APPROVED** [review]
LGTM.

* lint is 0.3 seconds slower on my laptop, that's acceptable
* Required doc changes don't seem too demanding based on previous PRs (and we can add ignores if it becomes annoying)

**Pull request merged**